hw/ip/usbdev/doc/wake_resume.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Suspending and Resuming with Sleep / Low Power Modes"
+---
```

Comment:
i might be slightly blind, but does this need to be linked in the main markdown anywhere?

Reply:
Might be, yeah -- I haven't become familiar with how we structure our docs.

Currently, it just creates a separate page underneath the usbdev section. But... the regref stuff doesn't look like it'll work :smile:

Reply:
Added an ability for regref to link to other pages, so this one links to the main USBDEV page.